draft-fossati-seat-expat.md

```diff
@@ -156,17 +156,27 @@ Endpoints that wish to receive attestation credentials using Exported Authentica
 The presence of this empty extension indicates that the requester understands this specification and is willing to process an attestation credential in the peer's Certificate message.
 
 An endpoint that supports this extension and receives a request containing it MAY include the cmw_attestation extension in its Certificate message, populated with attestation data. If the `cmw_attestation` extension appears in a Certificate message without it having been previously offered in the corresponding request, the receiver MUST abort the authenticator verification with an "unsupported_extension" alert. As specified in {{Section 9.3 of
-RFC8446}}, endpoints that do not recognize the cmw_attestation extension in a CertificateRequest or
+ -tls13}}, endpoints that do not recognize the cmw_attestation extension in a CertificateRequest or
 ClientCertificateRequest MUST ignore it and continue processing the message as if the extension were absent.
 
 ## Usage in Exported Authenticator-based Post-Handshake Authentication
 
 The `cmw_attestation` extension is designed to be used exclusively in Exported Authenticator-based post-handshake authentication as defined in {{RFC9261}}. It allows attestation credentials to be transmitted in the Authenticator's Certificate message only in response to an Authenticator Request. This ensures that attestation credentials are provided on demand rather than being included in the initial TLS handshake.
 
-To maintain a cryptographic binding between the Evidence and the authentication request, the `cmw_attestation` extension MUST be associated with the `certificate_request_context` of the corresponding CertificateRequest or ClientCertificateRequest message (from the Server or Client, respectively). This binding ensures that:
+To maintain a cryptographic binding between the Evidence and the authentication request, the `cmw_attestation` extension MUST be associated with the `certificate_request_context` of the corresponding CertificateRequest or ClientCertificateRequest message (from the Server or Client, respectively). This association ensures that the Evidence is specific to the authentication event.
 
-- The Evidence is specific to the authentication event and cannot be replayed across different TLS sessions.
-- The Evidence remains tied to the cryptographic context of the TLS session.
+## Cryptographic Binding of the Evidence to the TLS Session {#binding}
+
+The attester MUST bind the attestation evidence to the active TLS session. To do so, the attester derives a
```

Comment:
This is not a normative MUST, because it is built into the protocol. If you implement the protocol you get this property.